src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private File fileRead;
+    private File fileWrite;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        fileRead = new File(fromFileName);
+        fileWrite = new File(toFileName);
+        read(fileRead);
+        write(fileWrite);
```

Comment:
```suggestion
        String[] dataFromFile = read(fromFileName);
        String report = createReport(dataFromFile);
        write(report, toFileName);
```
your program should look like this